### ISSUE TITLE
More Control Over Custom Fees (for example, 1.5 sat/byte)

### ISSUE THREAD

**Issue by WordmanEric** [LedgerHQ/ledger-live-desktop#1363]
Users should be allowed to specify a non-whole number of satoshis per byte (for example, 1.5 sat/byte).  Obviously, once computed, the number would have to be rounded off to the nearest satoshi.

The electrum wallet has this feature, and I think it is extremely useful.

**Comment by gre** [comment]
@KhalilBellakrid  do you know if libcore will support this

**Comment by gre** [comment]
@WordmanEric definitely make sense.

**Comment by KhalilBellakrid** [comment]
Indeed, libcore is supporting only integer fees, we will have to change that !
